src/pages/index.js

```diff
@@ -174,15 +170,18 @@ function handleAddCardFormSubmit(cardData) {
     });
 }
 
-function handleProfileImageForm() {
+function handleProfileImageForm(inputValues) {
   //evt.preventDefault();
+  console.log(inputValues);
   profileImageModal.setLoading(true);
   const profileImage = document.querySelector(".profile__image");
   const profileImageInput = document.querySelector("#profile-image-link");
   profileImage.src = profileImageInput.value;
```

Comment:
You need to update `DOM` only in blocks `then` after a successful response from the server so that when there is an error it wouldn’t change